src/tusb_option.h

```diff
@@ -67,6 +67,9 @@
 #define OPT_MCU_STM32L1       308 ///< ST STM32L1
 #define OPT_MCU_STM32L4       309 ///< ST STM32L4
 
+#define OPT_MCU_MSP430x5xx    400 ///< TI MSP430x5xx

```

Comment:
the conflict is caused by the newly merged sony port which also use 400. Please increase this to 500 to resolve the conflict